pom.xml

```diff
@@ -10,7 +10,7 @@
   <parent>
     <groupId>org.jboss</groupId>
     <artifactId>jboss-parent</artifactId>
-    <version>39</version>
+    <version>46</version>
```

Comment:
I think we should try to target the version that WildFly is using: https://github.com/wildfly/wildfly/blob/main/pom.xml#L12